src/AirshipBindings.Android.ADM/build.gradle

```diff
@@ -27,7 +27,7 @@ task syncVersion  {
         def path = new File("$projectDir/Jars/$filename")
         if ( !path.exists() ) {
            // Download aar
-            def url = new URL("http://central.maven.org/maven2/com/urbanairship/android/urbanairship-adm/${airshipProperties.androidVersion}/$filename")
+            def url = new URL("https://repo1.maven.org/maven2/com/urbanairship/android/urbanairship-adm/${airshipProperties.androidVersion}/$filename")
```

Comment:
you should use the jcenter repos

Reply:
Maven Central no longer allows `http:`. This is the recommended URL. But Ryan wanted bintray, so I've changed it to bintray.